common/src/test/java/com/datastrato/gravitino/json/TestDTOJsonSerDe.java

```diff
@@ -361,6 +405,8 @@ public void testPartitioningDTOSerDe() throws Exception {
             .configure(EnumFeature.WRITE_ENUMS_TO_LOWERCASE, true)
             .build()
             .writeValueAsString(partitioning);
+    System.out.println("lmh");
+    System.out.printf("partitioning: %s\n", serJson);
```

Comment:
Remove these two lines, never do println in code.

Reply:
Apologies for forgetting to remove it, I have fixed it now.